src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -51,7 +51,7 @@ public override void VisitMarkupTextLiteral(MarkupTextLiteralSyntax node)
 
         public override void VisitMarkupLiteralAttributeValue(MarkupLiteralAttributeValueSyntax node)
         {
-            AddSemanticRange(node, RazorSemanticTokensLegend.MarkupAttributeValue);
+            AddSemanticRange(node, RazorSemanticTokensLegend.MarkupAttributeQuote);
```

Comment:
This is a hack. `markupAttributeValue` didn't have a mapping on the VS side, a PR for that is already merged but it won't  work until they insert again. Thankfully markupAttributeQuote and markupAttributeValue map to the same thing, so this is equivilant for VS.